src/components/AddressBox.tsx

```diff
@@ -59,7 +65,12 @@ const AddressBox = ({ addressHash, isSelected, onPress, ...props }: AddressBoxPr
     <AddressBoxStyled {...props} onPress={handlePress} style={[boxAnimatedStyle, props.style]}>
       <AddressBoxTop>
         <AddressBadgeStyled onPress={handlePress} addressHash={addressHash} textStyle={{ fontSize: 18 }} />
-        {isSelected && <Checkmark />}
+        <Group>
+          <AppText color="tertiary" size={14}>
+            group {address?.group}
+          </AppText>
+          {isSelected && <Checkmark />}
+        </Group>
```

Comment:
At least 2 people came to me asking for this feature https://github.com/alephium/mobile-wallet/issues/340

Their arguments made a lot of sense to me.

Reply:
Definitely.

Reply:
I've just seen that having the Checkmark next to the group screw up the vertical alignement (the group label is too high).

Reply:
Is it? I didn't notice that... Fixed ✅